app/controllers/api/v1/pizzas_controller.rb

```diff
@@ -1,6 +1,27 @@
 class Api::V1::PizzasController < ApplicationController
+  protect_from_forgery unless: -> { request.format.json? }
   def show
     selected_pizza = Pizza.find(params[:id])
-    render json: { pizza: selected_pizza, brand: selected_pizza.brand }
+    render json: {pizza: selected_pizza, brand: selected_pizza.brand, reviews: selected_pizza.reviews} 
   end
+
+  def new
+    pizza = Pizza.find(params[:id])
+    review = Review.new
+  end
+
+  def create
+    pizza_id = params[:pizza].to_i
```

Comment:
this line isnt needed, rails comes with type coercion even if this happens to be a string (and it should appear as an integer anyways)